stable/kommander/Chart.yaml

```diff
@@ -7,4 +7,4 @@ maintainers:
 - name: alejandroEsc
 - name: jimmidyson
 name: kommander
-version: 0.4.4
+version: 0.4.5
```

Comment:
i dont care too much, but should this be tied to KCL version? or is it coincidence?

Reply:
It's actually just coincedence... honest!